notebook/services/contents/tests/test_manager.py

```diff
@@ -528,3 +528,34 @@ def test_check_and_sign(self):
         cm.mark_trusted_cells(nb, path)
         cm.check_and_sign(nb, path)
         assert cm.notary.check_signature(nb)
+
+
+class TestContentsManagerNoAtomic(TestCase):
+
+    def setUp(self):
+        self._temp_dir = TemporaryDirectory()
+        self.td = self._temp_dir.name
+        self.contents_manager = FileContentsManager(
+            root_dir = self.td,
+        )
+        self.contents_manager.use_atomic_writing = False
+
+    def tearDown(self):
+        self._temp_dir.cleanup()
+
+    def new_notebook_no_atomic(self):
+        """
+
+        """
+        cm = self.contents_manager
+        model = cm.new_untitled(type="notebook")
+        name = model['name']
+        path = model['path']
+
+        full_model = cm.get(path)
+        nb = full_model['content']
+        nb['metadata']['counter'] = int(1e6 * time.time())
+        self.add_code_cell(nb)
+
+        cm.save(full_model, path)
+        return nb, name, path
```

Comment:
This is a 'smoketest' - a test that runs the code and will fail if it raises an error, but doesn't check what the code actually did.

Sometimes a smoke test is all we can do, but in this case I think we should go one better, and check that the file it should have created really exists.

It may be overkill for this case, but I have a package called `testpath` with functions for checking this kind of thing - e.g. [testpath.assert_isfile](http://testpath.readthedocs.org/en/latest/fsasserts.html#testpath.assert_isfile).